hackathon_site/dashboard/frontend/src/components/cart/CartSummary/CartSummary.tsx

```diff
@@ -55,7 +59,7 @@ const CartSummary = () => {
                     cartQuantity === 0 ||
                     cartOrderLoading ||
                     !teamSizeValid ||
```

Comment:
Should we allow test users to order hardware if they are not on a team of 2 and more?

Reply:
that is determined by the teamSizeValid variable and is based on what logistics decides so it'll change from year to year